library/src/main/java/com/novoda/downloadmanager/lib/DownloadReadyChecker.java

```diff
@@ -37,6 +37,7 @@ private boolean isDownloadManagerReadyToDownload(FileDownloadInfo downloadInfo)
         }
         switch (downloadInfo.getStatus()) {
             case 0: // status hasn't been initialized yet, this is a new download
+            case DownloadStatus.SUCCESS:
```

Comment:
So in a batch if one of the downloads has been successfully completed it caused the service to shut down? I'm just wondering why as there are clearly downloads in the queue as being pending :confused: 

Does this means that even if a download has successfully completed there is a chance to be processed again?